docs/services/data/_index.md

```diff
@@ -43,41 +43,47 @@ Or learn more about Viam's data management features, which can be broken down in
 
 ## On-device data management
 
-Manage data directly on your robot, with configurable data capture from various components, automated recovery after interruptions, and secure cloud synchronization for efficient data storage and management.
+The data management service:
+
+- Automatically resumes capturing data after your machine restarts.
+- Securely transfers data to the cloud at the frequency you define.
+- Automatically deletes local data after syncing for space management.
 
 {{< cards >}}
-{{% manualcard link="/services/data/capture/" %}}
+{{% manualcard link="/services/data/capture-sync/" %}}
 
-### Data capture
+### Data capture and cloud sync
 
-Configurable for various robot components like cameras, sensors, and motors, with configurable capture frequency.
-Automatically resumes after robot restarts.
+More details and how to configure capture and sync.
 
 {{% /manualcard %}}
-{{% manualcard link="/services/data/cloud-sync/" %}}
-
-### Cloud sync
+{{< /cards >}}
 
-Securely transfer data to the cloud at the frequency you define.
-Resilient to interruptions, and deletes local data post-sync for space management.
+You can configure the frequency of data capture individually for each supported component:
 
-{{% /manualcard %}}
+{{< cards >}}
+{{< relatedcard link="/components/arm/">}}
+{{< relatedcard link="/components/board/">}}
+{{< relatedcard link="/components/camera/">}}
+{{< relatedcard link="/components/encoder/">}}
+{{< relatedcard link="/components/gantry/">}}
+{{< relatedcard link="/components/motor/">}}
+{{< relatedcard link="/components/movement-sensor/">}}
+{{< relatedcard link="/components/sensor/">}}
+{{< relatedcard link="/components/servo/">}}
 {{< /cards >}}
 
 ## Cloud data management
 
-Experience streamlined data handling with advanced querying, viewing, and filtering capabilities, along with efficient data labeling and exporting tools.
-
-{{< cards >}}
-{{% manualcard link="/how-tos/sensor-data-query-with-third-party-tools/" %}}
-
-### Query data
+<br>
+{{<imgproc src="/architecture/data-flow.svg" resize="x1100" declaredimensions=true alt="Data flowing from local disk to cloud to the Viam app, SDKs, and MQL and SQL queries." >}}
+<br><br>
 
-Make SQL or MQL queries on synced sensor data, accessible through the Viam app and MQL-compatible clients.
+Experience streamlined data handling with advanced querying, viewing, and filtering capabilities, along with efficient data labeling and exporting tools.
 
 <!-- markdownlint-disable-file MD034 -->
 
-{{% /manualcard %}}
+{{< cards >}}
 {{% manualcard link="https://app.viam.com/data/view" %}}
```

Comment:
I don't love that this links to the app when everything else has docs associated with it.  Maybe the first thing shouldn't be a cards but should be mentioned before we list the cards?